src/main/java/com/fryrank/model/Review.java

```diff
@@ -1,32 +1,32 @@
 package com.fryrank.model;
 
-import lombok.Getter;
+import lombok.Data;
 import lombok.NonNull;
 import lombok.RequiredArgsConstructor;
 import org.springframework.data.annotation.Id;
-import org.springframework.data.mongodb.core.index.Indexed;
 import org.springframework.data.mongodb.core.mapping.Document;
 
 @Document("review")
 @RequiredArgsConstructor
```

Comment:
Same comment about `@Data`... we can remove the `@RequiredArgsConstructor`

Reply:
Done in [55c5367](https://github.com/NickPriv/FryRank/pull/10/commits/55c5367ea13efbcf23eacaf6e03235baa4982132).